scripts/generate-component.py

```diff
@@ -31,13 +31,7 @@ def generate_file(template, path, name, mapping):
     package = sys.argv[2]
 
 print('')
-print(f'Create compose component structure for {package}.Bpk{name}?')
-confirmation = input("[Y/n]: ").lower()
-if confirmation != "" and confirmation != "y" and confirmation != "yes":
-    sys.exit("Component creation cancelled")
-
-print('')
-print('Creating compose component structure...')
```

Comment:
I guess this is useful as a final sanity check before comitting it to make sure the name you entered is correct and no issues with spelling.

So would fall on the side of keeping it to make sure its a final confirmation to make sure all is ok with the naming :)